app/dashboard/templates/profiles/profile.html

```diff
@@ -22,72 +22,59 @@
     {% include 'shared/top_nav.html' with class='d-md-flex' %}
     {% include 'home/nav.html' %}
     <div class="container-fluid header profile-header dash" style="{% if profile.profile_wallpaper %} background-image: url( {{ profile.profile_wallpaper }} ) {% endif %}">
-        {% csrf_token %}
-        {% if is_editable %}
-        <div class="position-absolute profile-banner text-center">
-          <div class="h4 font-title-lg m-auto">
-            Edit Header Photo
-          </div>
+      {% csrf_token %}
+      {% if is_editable %}
+        <div class="position-relative profile-banner">
+		  <div class="container mt-5">
+		    <div class="row">
+			  <div class="col-12">
+				<span class="profile-banner-cta">
+			      <i class="fa fa-camera"></i>
+				  Update Header Image
+				</span>
+			  </div>
+			</div>
+		  </div>
         </div>
-          {% endif %}
-        {% if not hidden and is_staff %}
+      {% endif %}
+      {% if not hidden and is_staff %}
         <div class="alpha-warning font-caption mb-0">
           <span class="font-weight-bold">Staff only</span>
           <a style="color:white;" href="{{profile.admin_url}}">{% trans "Profile Admin" %}</a>
         </div>
       {% endif %}
     </div>
     <div class="container-fluid bg-light">
-        <div class="container profile-card mt-md-n5 mh-150">
-          <div class="row">
-            <div class="col-12 col-lg-6 profile-header__main-infos">
-              {% if not hidden %}
-                {% include 'profiles/header_avatar.html' %}
-                {% include 'profiles/header_details.html' %}
+      <div class="container profile-card mt-md-n5 mh-150">
+        <div class="row">
+          <div class="col-12 col-lg-6 profile-header__main-infos">
```

Comment:
there is some problem with if here "if not hidden is open 2 times and also some closed div inside, Lets be sure depending on the condition this will not let also orphan close divs.